skinning/json.md

```diff
@@ -0,0 +1,123 @@
+# skin.json
+This is the code that makes the skin work. For each keymode, you can see the column width, the hit position, and the column lighting tints.
+
+For each keymode, the format is as follows (using 7K as an example):
+
+```json
+"7k": {
+  "column_width": 80,
+  "hit_position": 144,
+  "colors": [
+    "#f7314f",
+    "#e85a5a",
+    "#f7314f",
+    "#dcc784",
+    "#f7314f",
+    "#e85a5a",
+    "#f7314f"
+  ]
+}
+```
+
+- `column_width` decides the width of all the columns in pixels.
+- `hit_position` decides how many pixels up from the bottom of the screen the hit position is.
+- `colors` tints the hit lighting of each individual column. The first entry will tint the lighting in the first lane from the left of the stage, the second entry will tint the second lane, and so on.
+
+This goes from 1K to 10K so all keymodes will need to be added.
```

Comment:
1-3k and 9-10k only are accessible thru osu!mania maps. And it doesn't really *need* to be added.